modules/registry-configuring-storage-azure-user-infra.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+//* registry/configuring-registry-storage-azure.adoc
+
+[id="registry-configuring-storage-azure-user-infra_{context}"]
+= Configuring registry storage for Azure
+
+During installation, your cloud credentials are sufficient to create Azure Blob
+Storage, and the Registry Operator automatically configures storage. Azure
+registry storage uses the `ImageRegistryConfigStorageAzure` struct.
```

Comment:
`ImageRegistryConfigStorageAzure` struct is in operator code, the customers have no way to see this in `configs.imageregistry.operator.openshift.io/cluster`, do we need remove this sentence？

Reply:
Thank you for the clarification! I will remove the sentence.